src/crypto/store/memory-crypto-store.ts

```diff
@@ -336,7 +336,11 @@ export class MemoryCryptoStore implements CryptoStore {
     // Olm Sessions
 
     public countEndToEndSessions(txn: unknown, func: (count: number) => void): void {
```

Comment:
I'd love an explanation of why this takes `txn` as an argument sometime :-)

Reply:
ugh. Because it wants to expose the same interface as `IndexeddbCryptoStore`. Which, for *some* methods, supports a transaction-based API.